firefox-ios/Client/Frontend/Settings/Main/Debug/ToggleCloseRemoteTabs.swift

```diff
@@ -0,0 +1,28 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+class ToggleCloseRemoteTabs: HiddenSetting, FeatureFlaggable {
```

Comment:
cc @cyndichin Since this might have some overlap with the proposed debug feature flag screen.

Reply:
thanks for tagging me @mattreaganmozilla! Ideally if this is purely for testing it can live within the feature flag section. However, I do not want that proposed screen to be a blocker for this PR and can refactor at a later date if warranted. 

@skhamis do you mind providing more context on the purpose of this setting (i.e. purely testing) and why we are using the different channels `.buildAndUser` and `.userOnly` for the `.remoteTabManagement` feature flag? Thanks!

Reply:
Hi @cyndichin! This feature will be released on Android & Desktop soon and overall the feature is still relatively new. The goal(s) for this setting is:
1. Control whether we broadcast the capability from iOS to other platforms
    - Ensures we don't unnecessarily launch this feature too early on iOS 
2. Eventually control whether we show the user that they can close tabs on other devices on iOS (this will be in a separate PR)
3. Ability to use nimbus to shutdown the feature if something goes wrong down the line 

Happy to make changes or use the new menus if they support the above cases!!

Reply:
@skhamis I have a draft for the PR, but the new screen is not ready yet. I will probably get to it later this week. Do you think we can hold off on adding this hidden setting until then? 

Otherwise, I would recommend we wrap this code for debug since our secret settings can be accessed in production and based on your notes it seems that we don't want users to access it. Also, the current capability is that nimbus is not necessarily shutting down the feature, this is changing the user default setting on a specific device.

Thanks! Let me know if you need more clarity.

Reply:
Definitely don't mind waiting to get this landed till next week if needed. This feature will have multi-parts so there's no rush landing this (though it'd be nice).

> Otherwise, I would recommend we wrap this code for debug since our secret settings can be accessed in production

I don't think we're necessarily concerned with users accessing it, the feature will work, it just won't have some of the polish we'll eventually be adding (notifications, etc). This is also a receive-only capability, however if we're still better off waiting I'll defer to the iOS team on this!

Reply:
Okay, if no concerns in exposing the feature, then no issue on my side in regards to the PR! 

We have some time before v130 gets released so can follow up with you on this when the debug feature flags PR gets merged. However, that new screen will be only available in beta or developer build. Are there concerns if we hide this work from users eventually?

Reply:
I don't think there's any concern to hide this, as long as developer/beta builds are able to toggle it (which sounds like it will be) because eventually we'll just flip the nimbus variable to be default on when we want to enable the feature. 

Appreciate looking into this and helping us out navigate the right away to go about feature flagging!

Reply:
Btw, the debug setting menu has been merged, if you would like to add your toggle there in this PR or a separate PR.

Reply:
@cyndichin I have updated it to use the new debug setting (I made it a separate commit so it's easier to review) -- seemed fairly straightforward and everything testing okay, but let me know if I missed anything!

Reply:
thanks! I left one comment that should be addressed, but otherwise looks great. thank you for making a separate commit :)